.docker/aiida-core-base/s6-assets/init/aiida-prepare.sh

```diff
@@ -5,34 +5,19 @@
 # Environment.
 export SHELL=/bin/bash
 
-# Configure AiiDA.
-export SETUP_DEFAULT_AIIDA_PROFILE=true
-export AIIDA_PROFILE_NAME=default
-export AIIDA_USER_EMAIL=aiida@localhost
-export AIIDA_USER_FIRST_NAME=Giuseppe
-export AIIDA_USER_LAST_NAME=Verdi
-export AIIDA_USER_INSTITUTION=Khedivial
-export AIIDA_PROFILE_PATH=/aiida/assets/config-quick-setup.yaml
-
 # Check if user requested to set up AiiDA profile (and if it exists already)
-if [[ ${SETUP_DEFAULT_AIIDA_PROFILE} == true ]] && ! verdi profile show ${AIIDA_PROFILE_NAME} &> /dev/null; then
-    NEED_SETUP_PROFILE=true;
-else
-    NEED_SETUP_PROFILE=false;
-fi
-
-# Setup AiiDA profile if needed.
-if [[ ${NEED_SETUP_PROFILE} == true ]]; then
+# If the environment variable `SETUP_DEFAULT_AIIDA_PROFILE` is not set, set it to `true`.
+if [[ ${SETUP_DEFAULT_AIIDA_PROFILE:-true} == true ]] && ! verdi profile show ${AIIDA_PROFILE_NAME} &> /dev/null; then
 
     # Create AiiDA profile.
     verdi quicksetup              \
         --non-interactive                            \
-        --profile "${AIIDA_PROFILE_NAME}"            \
-        --email "${AIIDA_USER_EMAIL}"                \
-        --first-name "${AIIDA_USER_FIRST_NAME}"      \
-        --last-name "${AIIDA_USER_LAST_NAME}"        \
-        --institution "${AIIDA_USER_INSTITUTION}"    \
-        --config "${AIIDA_PROFILE_PATH}"
+        --profile "${AIIDA_PROFILE_NAME:-default}"            \
+        --email "${AIIDA_USER_EMAIL:-aiida@localhost}"                \
+        --first-name "${AIIDA_USER_FIRST_NAME:-Giuseppe}"      \
+        --last-name "${AIIDA_USER_LAST_NAME:-Verdi}"        \
+        --institution "${AIIDA_USER_INSTITUTION:-Khedivial}"    \
+        --config /aiida/assets/config-quick-setup.yaml
```

Comment:
Why not make this also configurable?

Reply:
You mean the last one? It is the file for config (https://github.com/aiidateam/aiida-core/blob/main/.docker/aiida-core-base/s6-assets/config-quick-setup.yam) which contains things that I think usually fixed for a regular use of setting. 

It is copy to the container by https://github.com/aiidateam/aiida-core/blob/6ee278cebe8fb58cd6e69517d678b97570d0d661/.docker/aiida-core-base/Dockerfile#L166, so the file path makes no sense to be configurable, I assume.

Reply:
Yes, but the user may add a custom config file in a mounted volume and thereby customize the profile parameters, right?

Reply:
You are right. I added it back and add a note to the Doc.